src/Protocol/Diaspora.php

```diff
@@ -1858,36 +1852,10 @@ private static function receiveConversationMessage(array $importer, array $conta
 			'contact-id' => $contact['id'],
 			'title'      => $subject,
 			'body'       => $body,
-			'seen'       => 0,
```

Comment:
`seen` and `reply` information does not matter anymore?

Reply:
There are matching default values in the database.

Reply:
And those were not in place before? Just wondering ;-)

Reply:
I needn't to change the default values. BTW:I guess that this wasn't my last work on this whole stuff. By new nearly nothing had been centralized with mails.